android/app/src/main/kotlin/io/lantern/model/SessionModel.kt

```diff
@@ -208,541 +129,330 @@ class SessionModel(
                 } else {
                     throw IllegalArgumentException("No URL provided for webview")
                 }
-
             }
 
-            "trackUserAction" -> {
-                val props: Map<String, String> = mapOf("title" to call.argument("title")!!)
-                Plausible.event(
-                    call.argument("name")!!, url = call.argument("url")!!, props = props
+            "submitStripePayment" -> {
+                paymentUtils.submitStripePayment(
+                    call.argument("planID")!!,
+                    call.argument("email")!!,
+                    call.argument("cardNumber")!!,
+                    call.argument("expDate")!!,
+                    call.argument("cvc")!!,
+                    result
                 )
             }
 
-            "acceptTerms" -> {
-                LanternApp.getSession().acceptTerms()
-            }
+            "checkForUpdates" -> {
+                autoUpdater.checkForUpdates(result)
 
-            "setPaymentTestMode" -> {
-                LanternApp.getSession().setPaymentTestMode(call.argument("on") ?: false)
-                activity.restartApp()
             }
 
-            "setPlayVersion" -> {
-                LanternApp.getSession().isStoreVersion = call.argument("on") ?: false
-                activity.restartApp()
+            "isPlayServiceAvailable" -> {
+                result.success(LanternApp.getInAppBilling().isPlayStoreAvailable())
             }
 
-            "setForceCountry" -> {
-                LanternApp.getSession().setForceCountry(call.argument("countryCode") ?: "")
-                activity.restartApp()
-            }
+            else -> super.doOnMethodCall(call, result)
+        }
 
-            "submitFreekassa" -> {
-                val userEmail = call.argument("email") ?: ""
-                val planID = call.argument("planID") ?: ""
-                val currencyPrice = call.argument("currencyPrice") ?: ""
-                activity.startActivity(
-                    Intent(activity, FreeKassaActivity_::class.java).apply {
-                        putExtra("userEmail", userEmail)
-                        putExtra("planID", planID)
-                        putExtra("currencyPrice", currencyPrice)
-                    },
-                )
-            }
+    }
 
-            "checkForUpdates" -> {
-                autoUpdater.checkForUpdates()
```

Comment:
@atavism Could you make sure autoupdates work? and also updating to this version from old, do we get crash or not?